src/Education.UnitTests/Commands/RuleDescriptionWindowCommandTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿/*
```

Comment:
I took inspiration from the IssueVisualizationToolWindowCommandTests, I deleted the tests we do not need and adjusted the ones we do need. 

Sidenote: I deleted the IVsMonitorSelection because I do not quite understand what it does and it seems it is not needed for an empty implementation, please lmk if I need to put it back.

Reply:
Deleting the `IVsMonitorSelection` makes sense - we don't know if we will need it yet.